webpages/popup/index.js

```diff
@@ -48,21 +51,25 @@ const vue = new Vue({
       }
     },
     iframeSrc(addonId) {
+      // Problem here
+      if (addonId !== this.currentPopup._addonId) return;
```

Comment:
What does this solve?

Reply:
I forgot to move the `// Problem here` but I need to look more into it... iframeSrc sometimes gets called for no reason...

Reply:
Is this related to this PR...?